README.md

```diff
@@ -223,9 +223,9 @@ For each relevant `$di->params['ExampleForAutoResolution']` element that is miss
 
 ```php
 <?php
-$di->params['ExampleForAutoResolution']['foo'] = 'bar';
 $di->params['ExampleForAutoResolution']['baz'] = array();
 $di->params['ExampleForAutoResolution']['dib'] = $di->lazyNew('Example');
+$di->params['ExampleForAutoResolution']['foo'] = 'bar';
```

Comment:
I thought so, but I assumed you would prefer to keep the same order as in
the constructor.

--
Alejandro Celaya Alastrué
www.alejandrocelaya.com
El 5/4/2015 20:33, "Hari K T" <notifications@github.com> escribió:

> In README.md
> <https://github.com/auraphp/Aura.Di/pull/83#discussion_r27779406>:
>
> >  $di->params['ExampleForAutoResolution']['baz'] = array();
> >  $di->params['ExampleForAutoResolution']['dib'] = $di->lazyNew('Example');
> > +$di->params['ExampleForAutoResolution']['foo'] = 'bar';
>
> Even though this doesn't matter ;) .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/auraphp/Aura.Di/pull/83/files#r27779406>.
>